Project.toml

```diff
@@ -18,9 +18,9 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 BinaryProvider = "^0.5.3"
 CMake = "^1.1.1"
-CxxWrap = "^0.9.0"
+CxxWrap = "0.9,0.10"
```

Comment:
Is this really meant to be compatible with both? If so, why is the version of the package even changed from 0.3.3 to 0.4.0, and not just to 0.3.4?